modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/internal/notification/EmailNotificationSender.java

```diff
@@ -67,9 +67,11 @@ public void sendNotification(
 				notificationRecipient.getName());
 
 			String subject = NotificationTemplateRenderer.render(
-				notificationTemplateContext, NotificationField.SUBJECT);
+				notificationTemplateContext, NotificationField.SUBJECT,
+				NotificationTemplateRenderer.MODE_PLAIN);
 			String body = NotificationTemplateRenderer.render(
-				notificationTemplateContext, NotificationField.BODY);
+				notificationTemplateContext, NotificationField.BODY,
+				NotificationTemplateRenderer.MODE_HTML);
```

Comment:
We need additional context to determine when we should escape and when we don't need to escape. 

The "Subject" field, should be treated as plain text while the "Body" field is treated as HTML.